docs-src/concepts/what-is-a-workflow-execution.md

```diff
@@ -38,6 +38,16 @@ Scalability is responsiveness in the presence of load.
 A single Workflow Execution is limited in size and throughput but is scalable because it can [Continue-As-New](/concepts/what-is-continue-as-new) in response to load.
 A Temporal Application is scalable because the Temporal Platform is capable of supporting millions to billions of Workflow Executions executing concurrently, which is realized by the design and nature of the [Temporal Cluster](/concepts/what-is-a-temporal-cluster) and [Worker Processes](/concepts/what-is-a-worker-process).
 
+### Replays
+
+A Replay is effectively a retry of the Workflow Execution that checks against an existing Event History. Replays are necessary and often happen to give the effect that Workflow Executions are resumable, reliable, and durable.
```

Comment:
I would open with:

> A Replay is the method by which a Workflow Execution resumes making progress.
> During a Replay the Commands that are generated are checked against an existing Event History..

And then here https://docs.temporal.io/workflows#deterministic-constraints at the end of the sentence:

>The execution semantics of a Workflow Execution include the re-execution of a Workflow Function.

Add ", this is called a Replay." and link to the Replay section.